src/commands/deploy.js

```diff
@@ -296,6 +296,22 @@ const runDeploy = async ({
   }
 }
 
+const handleBuild = async ({ context, flags }) => {
+  if (flags.build) {
```

Comment:
[sand] Should we use an early exit to avoid indentation?

Reply:
Done in https://github.com/netlify/cli/pull/2593/commits/21f6f2805f084201a37b5a93244022f11ae1d0d5